helmfile/values/dex.yaml.gotmpl

```diff
@@ -39,6 +39,21 @@ config:
       clientSecret: {{ .Values.dex.googleClientSecret }}
       hostedDomains: {{ toYaml .Values.dex.allowedDomains | nindent 6 }}
   {{ end }}
+  {{ if eq .Values.dex.oidcProvider "okta" }}
+  - type: oidc
+    id: okta
+    name: Okta
+    config:
+      # Canonical URL of the provider, also used for configuration discovery.
+      # This value MUST match the value returned in the provider config discovery.
+      #
+      # See: https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfig
+      issuer: https://elastisys-trial.okta.com
```

Comment:
This should probably be read from as a configuration option.

Reply:
yep, would make sense - but for consistency I'll change that for the other providers too

Reply:
Yes, fix them both. Don't forget to update the readme in that case so that we wont trip over it when upgrading :)